ethcore/light/src/net/mod.rs

```diff
@@ -157,6 +158,54 @@ pub struct Peer {
 	awaiting_acknowledge: Option<(SteadyTime, Arc<FlowParams>)>,
 }
 
+/// Whether or not a peer was kept by a handler
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PeerStatus {
```

Comment:
Even if you can assume it, both would be reasonable choices and there's no way to know your assumption is correct without reading the code. If I can put all the information you need in the type signature then I will. We could use `u8` instead of `bool` and instead of type-safe enums like in C, but we don't because they are semantically different. Likewise here.